source/config/__init__.py

```diff
@@ -19,6 +19,10 @@
 import easeOfAccess
 import winKernel
 
+#Constants for indent tones and speech.
+INDENT_SPEECH = 1
+INDENT_TONES = 2

```

Comment:
nit: I totally get why you'd put these constants here, but I think I'd prefer to see them in the speech module. While we have to define the config spec here, I'd prefer to keep subsystem specific stuff with the subsystem as much as possible.